buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/plugins/performance/PerformanceTestPlugin.kt

```diff
@@ -438,6 +439,19 @@ class PerformanceTestPlugin : Plugin<Project> {
         }
     }
 
+    private
+    fun Project.configureForAnyDistributedPerformanceTestTask(task: DistributedPerformanceTest) {
```

Comment:
The fact that we have to do this trick tells us that our samples handling is a bit too complicated. Creating the samples should probably be done by the performance tests themselves, as necessary. Then we wouldn't have to do other tricks like `-x prepareSamples`. 

No action required here, just noticing that the code is trying to tell us something :)